config.json

```
@@ -331,6 +331,16 @@
         "Config.HeaderTab": {
             "label": "Config.headerTab",
             "formFields": {
+
+                "header.low_cls_mode": {
+                    "type": "checkBox",
+                    "required": false,
+                    "label": "Config.headerSticky",
+                    "options": {
+                        "defaultValue": "false"
```

```suggestion
                        "defaultValue": "true"
```
Let's activate the sticky header by default, so that new users can benefit from the improved CLS right away.